packages/compass-components/src/components/signal-popover.tsx

```diff
@@ -59,7 +60,11 @@ const signalCardContentStyles = css({
   paddingBottom: spacing[4],
   paddingLeft: spacing[4],
   paddingRight: spacing[4],
-  backgroundColor: palette.white,
+  backgroundColor: 'var(--signalCardBackgroundColor)',
```

Comment:
For my learning, what's the benefit of using css variables here instead of setting background to a different color in the `signalCardContentDarkModeStyles`? Is it intended to be used other places?

Reply:
It reads easier and cleaner from my perspective: I define the actual style only once and in one place and only switch custom property values when theme changes. We already use custom properties in other places. Does this make sense?

Reply:
Sounds good thanks